Database/Corrections/TBC/tbcQuestFixes.lua

```diff
@@ -729,6 +729,9 @@ function QuestieTBCQuestFixes:Load()
         [10231] = {
             [questKeys.triggerEnd] = {"Beat Down \"Dirty\" Larry and Get Information", {[zoneIDs.SHATTRATH_CITY]={{43.86,27.97},},}},
         },
+        [10264] = {
+            [questkeys.preQuestSingle] = {10552},
```

Comment:
Noted this too late for the merged PRs, but the variable is called `questKeys` not `questkeys`. You should grab a better IDE to show you these mistakes 😉 

```suggestion
            [questKeys.preQuestSingle] = {10552},
```